extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/extractors/YoutubeStreamExtractor.java

```diff
@@ -261,17 +212,14 @@ public Description getDescription() throws ParsingException {
     }
 
     @Override
-    public int getAgeLimit() throws ParsingException {
-        assertPageFetched();
-        if (!isAgeRestricted) {
+    public int getAgeLimit() {
+        if (initialData == null || initialData.isEmpty()) throw new IllegalStateException("initialData is not parsed yet");
```

Comment:
Why doesn't `assertPageFetched()` work anymore?

Reply:
Because getAgeLimit is (now indirectly) being called in `onFetchPage()`.